packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_navigation.spec.ts

```diff
@@ -321,6 +321,79 @@ context('cy.origin navigation', () => {
       })
     })
 
+    it('supports redirecting from secondary to primary outside of cy.origin', () => {
```

Comment:
`cy.origin` isn't needed here since the requested url resolves to the primary domain. Essentially, a request is sent to the backend to resolve the url which follows redirects and determines the url is actually for the primary domain. This information is sent back to the frontend which determines the redirected url is for the same origin and changes the iframe src appropriately.

Reply:
Ok that makes sense!